deluge/ui/gtkui/filtertreeview.py

```diff
@@ -335,7 +335,7 @@ def set_menu_sensitivity(self):
             item.set_sensitive(sensitive)
 
     def select_all(self):
-        'for use in popup menu'
+        # for use in popup menu
```

Comment:
This should be a docstring

Reply:
Yup sorry meant to make same change as above...whoever added these bits of code didn't flake8 shoulda realized that but really was wanting passing tests to see when i broke things :P